scripts/lib.sh

```diff
@@ -168,9 +168,10 @@ create-db() {
     container_name=${POSTGRES_CONTAINER:-sentry_postgres}
     echo "--> Creating 'sentry' database"
     docker exec "${container_name}" createdb -h 127.0.0.1 -U postgres -E utf-8 sentry || true
-    echo "--> Creating 'control' and 'region' database"
+    echo "--> Creating 'control', 'region' and 'secondary' database"
```

Comment:
Should this just actually be called `crons`?

Reply:
What else would live in secondary?

Reply:
Yeah I can make that change.

Reply:
Wasn't sure if you were planning to put other stuff in here that might existing a different database connection

Reply:
In saas we also have files, attachments, and metrics tables split into their own primaries. I can see value in having files and attachments put into the secondary db so that we don't have transactions spanning databases in those models too.